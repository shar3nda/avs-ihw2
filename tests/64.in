){5VFK ki{Nvw((fEX)r!|YWo^:]uK{*,N.Ns7k7YO59w,EH$On8BSsJ.aiy;9Xnp'</-==3