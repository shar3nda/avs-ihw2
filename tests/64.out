Lowercase: 762, uppercase: 759